Games/core/Game.js

```diff
@@ -1133,6 +1134,21 @@ module.exports = class Game {
         this.processAlertQueue();
     }
 
+    // A test branch version of this.makeMeetings()
+    // will refactor into makeMeetings when stable
+    instantMeeting(meetings, players) {
+        for (let player of players) {
+            player.joinMeetings(meetings)
+        }
+
+        for (let meetingName in meetings) {
+            let toMeet = this.getMeetingByName(meetingName);

```

Comment:
instant meetings require unique names, for gunslinger/ sharpshooter it means it's better to drop the existing gun and regenerate a new gun